paasta_tools/cli/cmds/logs.py

```diff
@@ -1332,7 +1333,11 @@ def paasta_logs(args: argparse.Namespace) -> int:
     if args.instances is None:
         instances = None
     else:
+
```

Comment:
I think you added newlines inadvertently in a couple places.  I don't think they're really helping readability or anything, so probably worth removing them.